src/main/resources/db/migration/V1_4__Team.sql

```diff
@@ -0,0 +1,9 @@
+create table  TEAM
```

Comment:
add al the changes in one script, no need to use 3

Reply:
resolved